docs/pages/management/operations/enroll-agent-into-automatic-updates.mdx

```diff
@@ -0,0 +1,226 @@
+```
+
+</TabItem>
+<TabItem label="systemd " options="Teleport Cloud">
+
+If you changed the agent user to run as non-root, create
+`/etc/teleport-upgrade.d/schedule` and grant ownership to your Teleport user.
+Else, you can skip this step:
+
+```code
+$ mkdir -p /etc/teleport-upgrade.d/
+$ touch /etc/teleport-upgrade.d/schedule
+$ chown <your-teleport-user> /etc/teleport-upgrade.d/schedule
+```
+
+Install the `teleport-ent-updater` package (note: your teleport agent will be restarted during install):
+
+```code
+$ apt install teleport-ent-updater
+# or
+$ yum install teleport-ent-updater
+```
+
+Finally, verify that the updater can see your version endpoint:
+
+```code
+$ teleport-upgrade dry-run
+```
+
+You should see one of the following messages, depending on the target version
+you are currently serving:
+
+```text
+no upgrades available (1.2.3 == 1.2.3)
+an upgrade is available (1.2.3 -> 2.3.4)
+```
+
+<Admonition type="note">
+`teleport-upgrade` may complain about not having a valid upgrade schedule.
+This is expected immediately after install as the maintenance schedule might
+not be exported yet.
+</Admonition>
+
+</TabItem>
+<TabItem label="teleport-kube-agent chart " options="Teleport Cloud">
+
+Add the following chart values to your existing agent `values.yaml`:
+
+```yaml
+updater:
+  enabled: true
+```
+
+Update the Helm chart release with the new values by running `helm upgrade`.
+
+You can validate the updater is running properly by checking if its pod is ready:
+
+```code
+$ kubectl get pods
+NAME                               READY   STATUS    RESTARTS   AGE
+my-agent-0                         1/1     Running   0          14m
+my-agent-1                         1/1     Running   0          14m
+my-agent-2                         1/1     Running   0          14m
+my-agent-updater-d9f97f5dd-v57g9   1/1     Running   0          16m
+```
+
+And by consulting its logs:
+
+```code
+$ kubectl logs <your-agent-release>-updater
+2023-04-28T13:13:30Z	INFO	StatefulSet is already up-to-date, not updating.	{"controller": "statefulset", "controllerGroup": "apps", "controllerKind": "StatefulSet", "StatefulSet": {"name":"my-agent","namespace":"agent"}, "namespace": "agent", "name": "my-agent", "reconcileID": "10419f20-a4c9-45d4-a16f-406866b7fc05", "namespacedname": "agent/my-agent", "kind": "StatefulSet", "err": "no new version (current: \"v12.2.3\", next: \"v12.2.3\")"}
+```
+
+</TabItem>
+</Tabs>
+
+## Next steps
+
+You can temporarily suspend automatic updates for an agent:
+
+<Tabs>
+<TabItem label="systemd">
+
+Disable the systemd timer:
+
+```code
+$ systemctl disable teleport-upgrade.timer
```

Comment:
I think we must pass the `--now` flag, otherwise it will keep running until systemd is stopped.

From systemctl manpage
```
Note that this command does not implicitly stop the units that are being
disabled. If this is desired, either combine this command with the --now switch, or invoke
the stop command with appropriate arguments later.
```